admin_manual/configuration/files/external_storage/auth_mechanisms.rst

```diff
@@ -39,6 +39,21 @@ does not work.
    (See :doc:`../../server/background_jobs_configuration`.) Be
    aware that the Ajax cron mode is triggered by browsing the ownCloud Web GUI.
 
+Known Limitations
+~~~~~~~~~~~~~~~~~
+
+Please be aware that because of the way that credentials are stored in ownCloud, any operations must be performed by the logged-in mount owner, because credentials are not stored anywhere.
+As a result, there are three known limitations.
+These are:
+
+#. Directly sharing the storage or any its sub-folders will fail. They will not appear to the recipient, because the mount cannot be set up due to missing credentials.
```

Comment:
Does this affect federation?

Reply:
@PVince81, @phil-davis can you clarify?

Reply:
I have not tried this directly myself, so not confident to advise words related to "local" and ""federated" sharing.

Reply:
The sharing action doesn't fail. It will go through but the recipient will not see mounted.

Federated sharing is affected as well because it works on "public link share token" basis (or something similar) which itself doesn't contain the user's storage password, so the storage cannot be mounted either.

I've raised https://github.com/owncloud/core/issues/32110 to hide the sharing option completely in the ext storage settings as it cannot work in such scenarios.